cmd/talosctl/cmd/talos/root.go

```diff
@@ -60,6 +62,18 @@ func WithClientNoNodes(action func(context.Context, *client.Client) error) error
 				opts = append(opts, client.WithContextName(Cmdcontext))
 			}
 
+			ctx = metadata.AppendToOutgoingContext(ctx, "runtime", "Talos")
```

Comment:
we should probably figure out a way to generalize this... probably need a call with @Unix4ever